tests/__tests__/sitka_utils.test.ts

```diff
@@ -1,13 +1,12 @@
-import { sitka } from "../sitka-test"
+import { sitkaFactory, sitka } from "../sitka-test"
 import { createAppStore } from "../../src/sitka"
 import rewire from "rewire"
 
 const utils = rewire("../../dist/sitka.js")
 
-const { text: textModule } = sitka.getModules()
-
 describe("Sitka Util Functions", () => {
   test(`createAppStore returns Redux store`, () => {
+    // todo: using sitka factory results in referenceError: alert is not defined.  Test will still pass
```

Comment:
Error presents itself when switching over to use sitkaFactory here. referenceError: alert not defined.

related to same error when we switch provideForks test to sitkaFactory (on diffferent branch)

Reply:
fixed